TriblerGUI/widgets/trustpage/js/style_config.js

```diff
@@ -70,7 +71,36 @@ var config = {
         strokeColor: "#333333"
     },
     radius_step: 120,
-    neighbor_level: 2
+    neighbor_level: 2,
+    help: {
+        label : {
+            fontSize: 10,
+            fontFamily: "sans-serif",
+            fontWeight: "bold",
+            color: "#FFFFFF",
+            offsetY: 10
+        },
+        examplePublicKey: "#bcb",
+        numberOfNodes: 5,
+        numberOfEdges: 3,
+        circleColor : {
+            nodeRadius: 20,
+            colorNames: ["Freerider", "Unreliable", "Neutral", "Reliable", "Trusted"]
+        },
+        circleSize : {
+            sizeNames: ["Least", "Less", "Average", "More", "Most"]
+        },
+        edgeWidth: {
```

Comment:
This is not just about the edgeWidth svg. It also contains some config fields for the edgeSeparator svg.